test/models/test_session_limitable.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class TestLimitable < ActiveSupport::TestCase
```

Comment:
```suggestion
class TestSessionLimitable < ActiveSupport::TestCase
```

Reply:
It will cause a superclass mismatch with integration test.